<launch>
	<node pkg="sbus_serial" type="sbus_serial_node" name="sbus_node">
		<param name="frame_id" value="base" />
		<param name="port" value="/dev/ttyTHS2" />
		<param name="refresh_rate_hz" value="5" />
		<param name="rxMinValue" value="172" />
		<param name="rxMaxValue" value="1811" />
		<param name="outMinValue" value="0" />
		<param name="outMaxValue" value="1000" />
		<param name="silentOnFailsafe" value="true" />
	</node>
</launch>
